src/applications/personalization/profile-2/components/Profile2Wrapper.jsx

```diff
@@ -90,7 +90,7 @@ const Profile2Wrapper = ({
         />
       </div>
 
-      <div className="vads-l-grid-container vads-u-padding-x--0">
+      <div className="vads-l-grid-container vads-u-padding-x--0 va-profile-2-wrapper">
```

Comment:
Needed to add this class to supply to the 2 nightwatch tests in this PR.

Reply:
if you made this `va-profile-wrapper` would the nightwatch tests need to be updated?

Reply:
There is some css that is dependent on the `va-profile-wrapper` class, I want to make sure those don't bleed over into profile 2, so I update the class name.